apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -61,7 +61,7 @@
 else:
   from django.utils.translation import ugettext as _
 
-LOG = logging.getLogger()
+LOG = logging.getLogger(__name__)
```

Comment:
@amitsrivastava I think this will regress the logger. Need to remove __name__

Reply:
+1